build/mbl/build-update-payloads.py

```diff
@@ -77,15 +77,18 @@ def main():
 
     # Create the payloads
     create_update_payload_commands = [
-        "create-update-payload -b1 -o {}/wks_bootloader1_payload.tar".format(
+        "create-update-payload -b1 -o {}/wks_bootloader1_payload.swu".format(
             args.outputdir
         ),
-        "create-update-payload -b2 -o {}/wks_bootloader2_payload.tar".format(
+        "create-update-payload -b2 -o {}/wks_bootloader2_payload.swu".format(
             args.outputdir
         ),
-        "create-update-payload -k -o {}/kernel_payload.tar".format(
+        "create-update-payload -k -o {}/kernel_payload.swu".format(
             args.outputdir
         ),
+        "create-update-payload -r {} -o {}/rootfs_payload.swu".format(
+            args.image, args.outputdir
```

Comment:
Where does args.image come from?

Reply:
Erm... the command line. I don't think I understand what you're getting at.

Reply:
What I mean is I think you need to add image to the _parse_args function.

Reply:
It's already there.

Reply:
It's already there.

Reply:
My mistake - looked at an older version.